pkg/compact/compact_test.go

```diff
@@ -125,3 +137,40 @@ func TestGroupMaxMinTime(t *testing.T) {
 	testutil.Equals(t, int64(0), g.MinTime())
 	testutil.Equals(t, int64(30), g.MaxTime())
 }
+
+func BenchmarkGatherNoCompactionMarkFilter_Filter(b *testing.B) {
+	ctx := context.TODO()
+	logger := log.NewLogfmtLogger(os.Stderr)
```

Comment:
Here we are kind of testing how fast is writing messages to the terminal. Maybe we could do:
```suggestion
	logger := log.NewLogfmtLogger(ioutil.Discard)
```
Here?